### DISCUSSION TITLE
continuous  dc5f0cf  Time Axis does not regress automatically after clicking CHARGE

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1564] [General]
Continuous build 557811e Later updated. Time Axis not auto-regressing 
![ca37e4ed8c43151ec449c75b30c1b76](https://github.com/artisan-roaster-scope/artisan/assets/149906183/42ff7448-2190-4e58-9c35-6a4d29402784)

**Comment by Terracotta-6** [comment]
It's fixed. Thanks.

**Comment by MAKOMO** [comment]
Excellent! Thanks for reporting!
